cmd/waku/flags_rln.go

```diff
@@ -73,5 +73,11 @@ func rlnFlags() []cli.Flag {
 				Value: &options.RLNRelay.MembershipContractAddress,
 			},
 		},
+		&cli.IntFlag{
+			Name:        "rln-relay-bandwidth-threshold",
+			Value:       0,
+			Usage:       "Message rate in bytes/sec after which verification of proofs should happen. Use 0 to disable bandwidth rate limits",
```

Comment:
I think you meant Message rage in bits/sec as bandwidth is always represented in bits/sec.

Reply:
I copy-pasted this text from nwaku :-P 
But then, checking how the bandwidth is verified, I use bytes there (len of a byte slice)

Reply:
This was more from a user-facing experience point of view. People are using to bits/sec when bandwidth is specified. We should keep the user-facing flag to use bits/sec and internally multiply by 8 to store the rate as it is easy to account. 

Let us ask the nwaku team also to follow the same.

Reply:
cc: @rymnc

Reply:
Hmm yes, we have to revert that commit. Based on our offsite discussion, we will have to move the bandwidth constraining into its own validator, before the rln-relay validator - I'll create an issue and link it here for the design, does that work?

And yes, we'll use bits/sec!

Reply:
Works for me. I'll keep the PR open and update it accordingly according to the design. Thank you!

Reply:
@rymnc i would say we also need to remove that flag and have it as a hardcoded parameter? otherwise if different nodes have different configuration a message can be valid for ones not not for others.